tests/lib/gcf_sms/src/main.c

```diff
@@ -172,9 +172,6 @@ ZTEST(gcf_sms, test_gcf_sms_cmss)
 {
 	int err;
 
-	/* This test require a SIM supporting SMS and mfw 1.3.2(?) or newer. */
-	Z_TEST_SKIP_IFNDEF(CONFIG_SIM_HAS_SMS_SUPPORT);
```

Comment:
@eivindj-nordic Looks like you've added this. Just wondering if this is used in testing? If this is still being used, the `CONFIG_` prefix probably needs to be removed.

Reply:
Good catch, I think all nodes running this test should have mfw >?= 1.3.2, though there might be SIMs in CI without SMS support. This test is likely not ran in CI ever, so perhaps it can be removed @nordic-hani.